cascade-merge/action.yml

```diff
@@ -0,0 +1,213 @@
+# more info on https://ledgerhq.atlassian.net/l/cp/Ar8F1nmM
+name: 'Cascade merge'
+description: 'Automatically cascade merge changes from one branch to others'
+branding:
+  icon: 'git-pull-request'
+  color: 'purple'
+
+inputs:
+  gh-token:
+    description: 'pass GITHUB_TOKEN'
+    required: true
+  default-branch:
+    description: 'Default branch name (e.g., `main` or `master`)'
+    required: false
+    default: 'main'
+  slack-webhook-url:
+    description: 'SLACK_WEBHOOK_URL'
+    required: false
+  slack-api-token:
+    description: 'SLACK_API_TOKEN'
+    required: false
+  auto-merge:
```

Comment:
Not sure it's useful, it look dangerous to skip the CI and all human validation

In which case we would want to have it `true` ?

Reply:
it's a feature requested by Meri, but by default is not used.